custom_dc/deploy_cloud.md

```diff
@@ -27,7 +27,7 @@ You push a locally built Docker image to the [Google Cloud Artifact Registry](ht
 
 This procedure creates a "dev" Docker package that you upload to the Google Cloud Artifact Registry, and then deploy to Google Cloud Run. 
 
-1. Build a local version of the Docker image, following the procedure in [Build the local repo](build_repo.md).
+1. Build a local version of the Docker image, following the procedure in [Build the local repo](/custom_dc/build_repo.html).
```

Comment:
To followup on the comment I left in the `data_cloud.md` file, this link works because you have a leading `/` in front of `custom_dc`. I'd choose one of `/custom_dc/build_repo.html` or `build_repo.html` and use the same for both.